encrypted-config-value/src/main/java/com/palantir/config/crypto/Base64Utils.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2015 Palantir Technologies, Inc. All rights reserved.
+ */
+
+package com.palantir.config.crypto;
+
+import java.util.Base64;
+
+public final class Base64Utils {
+
+    private Base64Utils() {
+        /* do not instantiate */
+    }
+
+    public static boolean isBase64(String value) {
```

Comment:
So, after a bunch of poking, I'm going to keep the current method of checking (try to decode) but taking the suggestion of throwing when invalid. The spec is vague around how to deal with leftover lower-order bits/there are various edge cases the stackoverflow regexes miss.